src/schemas/__tests__/Entity.test.js

```diff
@@ -147,4 +147,13 @@ describe(`${schema.Entity.name} denormalization`, () => {
 
     expect(denormalize(1, menuSchema, entities)).toMatchSnapshot();
   });
+  it('can denormalize previously normalized data', () => {
```

Comment:
Sorry about misinterpretation. Figured it out when testing Union schema.

I'm closing this pull request.